lib/browser.ts

```diff
@@ -937,15 +931,15 @@ export class ProtractorBrowser extends AbstractExtendedWebDriver {
             }
           }
         })
-        .then(() => {
+        .then(async () => {
```

Comment:
Can we async/await this promise chain?

Reply:
This was done in my previous 95 file PR.

Reply:
Fixed.